charts/kubernetes-log-collector/README.md

```diff
@@ -0,0 +1,180 @@
+
+In the case of `k3s`, the `kube-apiserver` component is built in to the same service binary as other components. The binary still provides a way to pass eqivalents of the command line parameters to the component; see the [`k3s` documentation](https://docs.k3s.io/security/hardening-guide#api-server-audit-configuration) for details.
+
+Regardless of how the service and its configuration are managed, you will need to check the same set of configuration parameters:
+1. `audit-policy-file`: Path to the file that defines the audit policy configuration
+2. `audit-log-path`: When set, requests to the API server are logged to this fully-qualified file path according to policy
+
+You will need to provide (or possibly edit, if one already exists) a policy file and make sure it is located where the `audit-policy-file` parameter is looking for it. If `kube-apiserver` is deployed via a static pod, you will probably want to provide the file and a log directory to the pod via `volumeMount` parameters.
+
+The chart is not concerned with the policy file, but you will need to note the value you set for `audit-log-path` (along with how that corresponds to any `volumeMount` directory mapping) to set the `directories.log` chart variable (the directory component of the path, mapped to the host filesystem) and the `config.log_file` chart variable (the base name component of the path).
+
+**Audit log policy considerations**
+
+It is unwise to log the full details of every API transaction, so the policy configuration allows you to specify rules by which you can get extra details in some situations and little-to-none in others.
+
+TODO: Describe what details are most relevant, link to example config files
+
+### Outbound network connectivity requirements
+* 35.188.42.15 (tcp/443)
+* 34.120.195.249 (tcp/443)
+* https://cwp-ingest.redcanary.io
+
+Note: cwp-ingest.redcanary.io IPs are static.
+
+To utilize a HTTP proxy, set the following value during your installation:
+```console
+--set config.http_proxy="https://HOST:PORT"
+```
+## Installation
+
+### Create a namespace for your installation
+
+```console
+kubectl create namespace <NAMESPACE_NAME>
+```
+
+### Create a secret to hold your private registry credentials
+
+The image pull secret is used to securely authenticate and authorize container image pulls from your private container registry. *This may not be required in all environments.*
+
+There are two options for doing this.
+
+The first method creates the secret by passing the credential information on the command line. Ensure the namespace parameter matches the one you just created.
+
+```console
+kubectl create secret docker-registry <SECRET_NAME> \
+  --docker-server=<YOUR_REGISTRY_SERVER> \
+  --docker-username=<YOUR_USERNAME> \
+  --docker-password=<YOUR_PASSWORD> \
+  --docker-email=<YOUR_EMAIL> \
```

Comment:
I know we talked about the email param yesterday - were you running in to issues pulling when it was omitted, or were you unable to create the secret entirely? I was at least able to create the secret on k3s.